src/biogeophys/Wateratm2lndType.F90

```diff
@@ -244,14 +244,14 @@ subroutine Restart(this, bounds, ncid, flag)
     !------------------------------------------------------------------------
 
 
```

Comment:
Commented out rather than deleting because this was the only variable in the subroutine; however, we may choose to remove the subroutine entirely.

Reply:
Yes, if we remove this, we should go ahead and just remove this whole subroutine.

As with the dyn root variable: If your main confidence on this one comes from the test suite, as opposed to tracing the variable through the code, then I'd be more comfortable if you also ran a longer test that covers flooding: a longer version of `ERS_D_Ld5.f10_f10_musgs.I2000Clm50BgcCropRtm.cheyenne_intel.rtm-rtmOnFloodOnEffvelOn` (e.g., maybe an ERS_Lm3 version of that test).

Reply:
Ok, removed.

As I did for root_depth, I tried your suggested test and got:
PASS